# 飞镖发射

Header header

# 发射队伍
# 红方
uint8 GROUP_RED = 1
# 蓝方
uint8 GROUP_BLUE = 2
uint8 group

# 发射时的剩余比赛时间
duration remainingTime